integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/client/ElasticsearchClientFactoryImplElasticsearchJavaIT.java

```diff
@@ -875,6 +875,11 @@ void authentication() {
 
 		String payload = "{ \"foo\": \"bar\" }";
 
+		wireMockRule1.stubFor( post( urlPathMatching( "/myIndex/myType/_search" ) )
+				.withRequestBody( equalToJson( payload ) )
+				.willReturn( elasticsearchResponse().withStatus( 401 )
+						.withHeader( "www-authenticate", "Basic realm=\"IT Realm\"" ) ) );
+
```

Comment:
this one might be interesting to know 🙂 `elasticsearch-java` client does not support "preemptive authentication" so it doesn' send the credentials right away, and it was causing the test to fail as the client never receied a 401 to send credentials.